app/src/main/java/com/android/voyageur/model/trip/TripsViewModel.kt

```diff
@@ -181,6 +191,50 @@ open class TripsViewModel(
     }
   }
 
+  open fun getPhotosForSelectedTrip(): List<String> {
+    return selectedTrip.value?.photos ?: emptyList()
+  }
+
+  open fun addPhotoToTrip(photo: String) {
+    if (selectedTrip.value != null) {
+      val trip = selectedTrip.value!!
+      val updatedTrip = trip.copy(photos = trip.photos + photo)
+      updateTrip(
+          updatedTrip,
+          onSuccess = {
+            /*
+                This is a trick to force a recompose, because the reference wouldn't
```

Comment:
I think selecting a null trip may no longer be needed as we fixed that by refactoring the equals method.

Reply:
I removed the line and tested the behavior and it works, so you're right.